### ISSUE TITLE
Trying to load Optifine causes OpenAL to not be loaded on launch with newest(?) Forge version (1.12.2 - 14.23.4.2759)

### ISSUE THREAD

**Issue by TheRektafire** [sp614x/optifine#1759]
Relevant log stuff:

`[03:08:42] [Sound Library Loader/INFO] [net.minecraft.client.audio.SoundManager]: Starting up SoundSystem...
[03:08:42] [Thread-5/INFO] [net.minecraft.client.audio.SoundManager]: Initializing LWJGL OpenAL
[03:08:42] [Thread-5/INFO] [net.minecraft.client.audio.SoundManager]: (The LWJGL binding of OpenAL.  For more information, see http://www.lwjgl.org)
[03:08:42] [Thread-5/ERROR] [net.minecraft.client.audio.SoundManager]: Error in class 'LibraryLWJGLOpenAL'
[03:08:42] [Thread-5/ERROR] [net.minecraft.client.audio.SoundManager]: Unable to initialize OpenAL.  Probable cause: OpenAL not supported.
[03:08:42] [Thread-5/WARN] [net.minecraft.client.audio.SoundManager]: ERROR MESSAGE:
[03:08:42] [Thread-5/INFO] [net.minecraft.client.audio.SoundManager]: Could not locate OpenAL library.
[03:08:42] [Thread-5/WARN] [net.minecraft.client.audio.SoundManager]: STACK TRACE:
[03:08:42] [Thread-5/INFO] [net.minecraft.client.audio.SoundManager]: org.lwjgl.openal.AL.create(AL.java:156)
[03:08:42] [Thread-5/INFO] [net.minecraft.client.audio.SoundManager]: org.lwjgl.openal.AL.create(AL.java:102)
[03:08:42] [Thread-5/INFO] [net.minecraft.client.audio.SoundManager]: org.lwjgl.openal.AL.create(AL.java:206)
[03:08:42] [Thread-5/INFO] [net.minecraft.client.audio.SoundManager]: paulscode.sound.libraries.LibraryLWJGLOpenAL.init(LibraryLWJGLOpenAL.java:164)
[03:08:42] [Thread-5/INFO] [net.minecraft.client.audio.SoundManager]: paulscode.sound.SoundSystem.CommandNewLibrary(SoundSystem.java:1576)
[03:08:42] [Thread-5/INFO] [net.minecraft.client.audio.SoundManager]: paulscode.sound.SoundSystem.CommandQueue(SoundSystem.java:2572)
[03:08:42] [Thread-5/INFO] [net.minecraft.client.audio.SoundManager]: paulscode.sound.CommandThread.run(CommandThread.java:121)
[03:08:42] [Sound Library Loader/WARN] [net.minecraft.client.audio.SoundManager]: ERROR MESSAGE:
[03:08:42] [Sound Library Loader/INFO] [net.minecraft.client.audio.SoundManager]: Could not locate OpenAL library.
[03:08:42] [Sound Library Loader/INFO] [net.minecraft.client.audio.SoundManager]: Starting up SoundSystem...
[03:08:42] [Thread-7/INFO] [net.minecraft.client.audio.SoundManager]: Switching to No Sound
[03:08:42] [Thread-7/INFO] [net.minecraft.client.audio.SoundManager]: (Silent Mode)
[03:08:43] [Sound Library Loader/INFO] [net.minecraft.client.audio.SoundManager]: Sound engine started
`

Basically for some reason the launcher can't find OpenAL when doing initialization whenever I try to use the newest version of Optifine from the optifine site with the newest Forge version from the Forge site, 1.12.2 - 14.23.4.2759. I can confirm it is optifine because I only have 3 mods at the moment and whenever I have optifine in the mods folder and launch MC it causes it to have no sound. Pressing F3+T fixes it when in-game but my laptop is pretty low spec so it takes a few seconds to reload everything which isn't particularly ideal.
